client/src/locales/en-US.json

```diff
@@ -21,6 +21,9 @@
         "clientRevoked": "You have been revoked from this organization. You can still see some of the data you had access to, but most features of Parsec will not work.",
         "expiredOrganization": "This organization has expired. Please contact an administrator."
     },
+    "formatter": {
+        "version": "v`{version}` "
```

Comment:
Why the backquotes around? It result in the display being ``v`3.1.0`‎ `` instead of just `v3.1.0`.